index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Create Mobile Version</title>
+</head>
+<body>
+    <main>
+        
+        <header>
+            <h1 class="logo">Welcome</h1>
+     
+            <nav class="menu-bar">
+                <div class="bar"></div>
+                <div class="bar"></div>
+                <div class="bar"></div>
+                <div class="bar"></div>
+            </nav>
+
+        </header>
+
+        <div class="background-image">
```

Comment:
- [ ] You should use `<section>` instead of `<div>` because it is semantically more appropriate for this section.

Reply:
The appropriate tag has been used as suggested